srclist_by_beam.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python 3
```

Comment:
Surely there shouldn't be a space between "python" and "3"!  Either way, the argument to env *should not* be python3, because some people still might be using 2, or the python3 executable is named "python".

Reply:
Definitely shouldn't be a space so I'll fix that.

I've found that when running on Galaxy you have to use python3 (unless there's some extra step I'm missing). That being said, if we install it using `python3 setup.py --build-scripts <build_locations>` the shebang will automatically change to where the python3 executable is located so I'm happy to leave it as python.